### DISCUSSION TITLE
Flex sizes. Fill available width and height equally

### DISCUSSION THREAD

**Discussion by Aunmag** [emilk/egui#878] [Q&A] [answered]
How do I make a set of buttons to fill the container/row by some relative proportion? Or can I do it at least for width only?

![Untitled](https://user-images.githubusercontent.com/18663542/140411983-357c04cf-7900-4423-995e-48972e79c7aa.png)

**Comment by emilk** [comment, marked as answer]
This is not currently possible

**Reply by Aunmag** [reply]
Сan I at least put two buttons horizontally so they fill the space equally?

**Reply by parasyte** [reply]
You can if you know how much space you want to allocate, and assuming you are ok with providing every rect size.

**Reply by BctfN0HUK7Yg** [reply]
``` rust
ui.columns(2, |columns| {
    columns[0].label("First column");
    columns[1].label("Second column");
});
```

**Reply by Aunmag** [reply]
@BctfN0HUK7Yg thanks buddy
